wonderland/src/index.js

```diff
@@ -95,25 +93,14 @@ export class ZestyBanner extends Component {
       import(networkingLink)
         .then(value => {
           this.zestyNetworking = Object.assign({}, value);
-          this.startLoading();
-          if (isBeta) {
-            setInterval(this.startLoading.bind(this), AD_REFRESH_INTERVAL);
-          }
         })
         .catch(() => {
           console.error('Failed to dynamically retrieve networking code, falling back to bundled version.');
           this.dynamicNetworking = false;
-          this.startLoading();
-          if (isBeta) {
-            setInterval(this.startLoading.bind(this), AD_REFRESH_INTERVAL);
-          }
         });
-    } else {
-      this.startLoading();
-      if (isBeta) {
-        setInterval(this.startLoading.bind(this), AD_REFRESH_INTERVAL);
-      }
     }
+    this.startLoading();
+    setInterval(this.startLoading.bind(this), AD_REFRESH_INTERVAL);
```

Comment:
Can we drop the refresh interval to 15 sec while we're at it?

Reply:
Might also be better to import this refresh interval globally from `networking.js`